src/main/java/org/radarbase/output/kafka/KafkaStorageFactory.kt

```diff
@@ -0,0 +1,27 @@
+package org.radarbase.output.kafka
+
+import io.minio.MinioClient
+import org.apache.hadoop.fs.FileSystem
+import org.radarbase.output.config.ResourceConfig
+import org.radarbase.output.config.ResourceType
+import java.nio.file.Path
+
+class KafkaStorageFactory(private val resourceConfig: ResourceConfig, private val tempPath: Path) {
```

Comment:
Why not name these as Source? like `SourceStorageFactory` and other source-related classes can be renamed too.